### DISCUSSION TITLE
MyBMW stuck connecting to IDrive4

### DISCUSSION THREAD

**Discussion by himlastar** [BimmerGestalt/AAIdrive#831] [General]
I am stuck here on my BMW F10 NBT 2013-12

!
![PXL_20241031_142542488](https://github.com/user-attachments/assets/ffd3ae17-3e6c-4c78-85f7-c3becf9d6f70)

I have My BMW app and it's connected but it doesn't work?

**Comment by hufman** [comment]
It seems that MyBMW is less reliable about the USB connection. Please try uninstalling it and installing BMW Connected 6.4, which should work well for establishing the car connection (even if the Connected app shows an error when you try to open it)

**Reply by himlastar** [reply]
It connects now, shows the apps and the disconnects, I have to reconnect the cable and it does it again. Any ideas maybe some android settings ? I used the 6.4 version and deleted MyBmw

**Reply by himlastar** [reply]
I took different usb cable and disabled adaptive battery and it work
![Uploading PXL_20241104_110946684.jpg…]()
s now, how can I mirror my screen?

**Reply by himlastar** [reply]
Okay I found where can I get that apk, now it screen mirrors but really low FPS, is it possible to optimise FPS ?

**Reply by hufman** [reply]
No, the screen mirrors by taking a screenshot of the phone and sending the screenshot to the car as fast as possible (unless you are driving, which is capped to 1fps for safety).

**Comment by harryberlin** [comment]
In my case, after every umounted and mount the phone, i have to wait for car shutdown. after this the apps are available.
don't know why, but this is my process.
F11 NBT Next
